azimuth_capi/operator.py

```diff
@@ -1315,17 +1313,9 @@ async def monitor_cluster_services(name, namespace, **kwargs):
                     "labels", {}
                 ):
                     cluster_services[service_name] = service_status
-            await ekclusterstatus.json_patch(
-                name,
-                [
-                    {
-                        "op": "replace",
-                        "path": "/status/services",
-                        "value": cluster_services,
-                    },
-                ],
-                namespace=namespace,
-            )
+            instance.status.services = cluster_services
+            await save_cluster_status(instance)
+            await update_platforms_for_cluster(instance)
```

Comment:
Can we try keeping just this line in here? I think we only need the instance here to create the owner reference.